Ports/kakoune/package.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=kakoune
+version=674053935d3260dd54c4e0084c76e9170d731eb2
+files="https://github.com/mawww/kakoune/archive/${version}.tar.gz ${version}.tar.gz daedcb1222a9c430a5760dcd0c66800fc45c76b450a5c08ac575daab3b022172"
+auth_type=sha256
```

Comment:
Please use quotes (single or double when you need interpolation) around all string values